### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by JasonPKnoll**
What we Did - 
- Added initial gems

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -37,14 +39,29 @@ gem 'bootsnap', '>= 1.1.0', require: false
...
   gem 'web-console', '>= 3.3.0'
   gem 'listen', '>= 3.0.5', '< 3.2'
+  gem 'rubocop-rails'
 end
 
+group :test do
+  gem 'capybara'
+  gem 'launchy'
+  gem 'simplecov'
+  gem 'vcr'
```
Comment (marlitas) [inline Gemfile]: Might not want to add vcr till we get backend endpoints set up?

Comment (kabcouwer) [inline Gemfile]: I don't think we are using vcr due to our endpoint limit of 1000. Sticking with webmock

**Review by marlitas — CHANGES_REQUESTED** [review]
Taking out VCR till we're ready for it.

**Comment by kabcouwer** [comment]
Looks good! Just need to remove 'vcr' gem and add the shoulda matchers setup in rails_helper:
https://github.com/thoughtbot/shoulda-matchers

**Review by kabcouwer — CHANGES_REQUESTED** [review]
Looks good! Just need to remove 'vcr' gem and add the shoulda matchers setup in rails_helper:
https://github.com/thoughtbot/shoulda-matchers

**Comment by JasonPKnoll** [comment]
removed vcr gem

**Pull request merged**